native/explorer/src/lazyframe.rs

```diff
@@ -344,3 +344,14 @@ pub fn lf_concat_columns(
 
     Ok(ExLazyFrame::new(out_df.drop([id_column])))
 }
+
+#[rustler::nif]
+pub fn lf_sql(lf: ExLazyFrame, sql_string: &str, table_name: &str) -> ExLazyFrame {
+    let mut ctx = polars::sql::SQLContext::new();
+
+    let lf = lf.clone_inner();
+    ctx.register(table_name, lf);
+
+    let lf_sql = ctx.execute(sql_string).unwrap();
```

Comment:
I think it's fine for this first version, but would be nice to treat the error and return a `Result<ExLazyFrame, ExplorerError>` for this function.

Reply:
I'd be fine doing it on this version too. Any special considerations you can think of for error handling?

Reply:
No, the idea is just to avoid unwrap panics :)

Reply:
[1882109](https://github.com/elixir-explorer/explorer/pull/918/commits/188210988b7c13e3d95e12be405c3cb9ef721cff)